ui/src/app/pages/admin/admin-institution.tsx

```diff
@@ -0,0 +1,153 @@
+import {Component} from '@angular/core';
+import {Button} from 'app/components/buttons';
+import {FadeBox} from 'app/components/containers';
+import {SemiBoldHeader} from 'app/components/headers';
+import {ClrIcon} from 'app/components/icons';
+import {institutionApi} from 'app/services/swagger-fetch-clients';
+import colors from 'app/styles/colors';
+import {ReactWrapperBase} from 'app/utils';
+import {reactStyles} from 'app/utils';
+import {Institution} from 'generated/fetch';
+import {DuaType, OrganizationType} from 'generated/fetch';
+import * as fp from 'lodash/fp';
+import {Column} from 'primereact/column';
+import {DataTable} from 'primereact/datatable';
+import * as React from 'react';
+
+
+const styles = reactStyles({
+  pageHeader: {
+    fontSize: '18px',
+    lineHeight: '22px',
+    marginBottom: '1rem',
+    marginTop: '0.5rem'
+  },
+  header: {
+    fontSize: '14px',
+    fontWeight: 600,
+    letterSpacing: 0,
+    lineHeight: '18px',
+    color: colors.primary,
+    width: '3rem',
+    textAlign: 'start',
+    height: '4rem',
+  },
+  text: {
+    fontSize: '14px',
+    letterSpacing: 0,
+    lineHeight: '18px',
+    color: colors.primary,
+    verticalAlign: 'top',
+    textOverflow: 'ellipsis',
+    overflow: 'auto',
+    height: '4rem',
+    width: '5rem',
+  }
+});
+
+interface State {
+  loadingInstitutions: boolean;
+  institutions: Array<Institution>;
+  institutionLoadError: boolean;
+}
+
+export class AdminInstitution extends React.Component<{}, State> {
+  constructor(props) {
+    super(props);
+    this.state = {
+      loadingInstitutions: true,
+      institutions: [],
+      institutionLoadError: false
+    };
+  }
+
+  async componentDidMount() {
+    try {
+      const details = await institutionApi().getInstitutions();
+      this.setState({
+        loadingInstitutions: false,
+        institutions: details.institutions
+      });
+    } catch (e) {
+      this.setState({
+        loadingInstitutions: false,
+        institutionLoadError: true
+      });
+    }
+  }
+
+  renderOrganizationType(row, col) {
+    switch ( row['organizationTypeEnum']) {
+      case OrganizationType.ACADEMICRESEARCHINSTITUTION: return 'Academic Research Institution';
+      case OrganizationType.HEALTHCENTERNONPROFIT: return 'Health Center Non-Profit';
+      case OrganizationType.EDUCATIONALINSTITUTION: return 'Educational Institution';
+      case OrganizationType.INDUSTRY: return 'Industry';
+      default: return 'Other';
```

Comment:
If organization type is OTHER, we should display the organizationTypeOtherText here instead.  Maybe both, something like "Other: WhateverSpecialType"